LORIS_indepth.html

```diff
@@ -697,7 +634,8 @@ <h2>LORIS - <font color="blue"> Links</font></h2>
 			<p>Source code: <a href="https://github.com/aces/Loris" target="_blank"> https://github.com/aces/Loris</a></p>
```

Comment:
line above:  put the actual loris-dev email address on a new line (it wraps awkwardly)

Reply:
Cool - I added a break to each line to keep it consistent